package.json

```diff
@@ -146,6 +146,18 @@
           "default": "haskell-language-server",
           "description": "Which language server to use."
         },
+        "haskell.languageServerReleasesPath": {
+          "scope": "resource",
+          "type": "string",
+          "default": "https://api.github.com/repos/haskell/haskell-language-server/releases",
```

Comment:
Can we make the default "", and then inside the extension use "https://api.github.com/repos/haskell/haskell-language-server/releases" if it is set to ""